app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+  has_many :users, through: :likes
+
+  after_save :update_posts_counter
+
```

Comment:
### SUGGESTION

- [ ] It would be better to make the `update_post_counter` method private 👍🏿